documentation/modules/ROOT/pages/02-basic-fundas.adoc

```diff
@@ -104,7 +104,7 @@ include::ROOT:partial$invoke-service.adoc[tag=*]
 :k8s-cli: oc
 include::ROOT:partial$invoke-service.adoc[tag=*]
 
-The last http command should return a response containing a line similar to **Hi  greeter => '6fee83923a9f' : 1**
```

Comment:
The last command in that partial above is not the `http` command, so it was causing confusion.